src/Host/Controllers/Identity/UsersController.cs

```diff
@@ -110,11 +110,12 @@ public Task<string> ForgotPasswordAsync(ForgotPasswordRequest request)
     }
 
     [HttpPost("reset-password")]
+    [AllowAnonymous]
     [OpenApiOperation("Reset a user's password.", "")]
     [ApiConventionMethod(typeof(FSHApiConventions), nameof(FSHApiConventions.Register))]
-    public Task<string> ResetPasswordAsync(ResetPasswordRequest request)
+    public Task<string> ResetPasswordAsync(ResetPasswordRequest request, [FromQuery] string tenant)
     {
-        return _userService.ResetPasswordAsync(request);
+        return _userService.ResetPasswordAsync(request, tenant);
```

Comment:
there's no need to pass the tenant to the service... The tenant is automatically picked up (and available) by the multitenant middleware:
https://github.com/fullstackhero/dotnet-webapi-boilerplate/blob/370dc7bb7a5c84af53ea37cdffa109060b07807e/src/Infrastructure/Multitenancy/Startup.cs#L28